app/features/build_runner/fastlane_build_runner.rb

```diff
@@ -76,8 +76,8 @@ def run(new_line_block:, completion_block:)
         return
       end
 
-      fast_file = Fastlane::FastFile.new(fast_file_path)
       FastlaneCore::Globals.verbose = true
+      ENV["FASTLANE_SKIP_DOCS"] = true.to_s
```

Comment:
This would otherwise generate the new `README.md` in the `./fastlane` folder, we don't need that as part of CI, ever 👍

Reply:
true